dandiapi/api/views/robots.py

```diff
@@ -0,0 +1,10 @@
+from django.views.decorators.cache import cache_page
+from django.http import HttpResponse
+
+@cache_page(60 * 60 * 24)  # 24 hour cache
+def robots_txt_view(request):
+    content = """
+    User-agent: *
+    Disallow: /
+    """
+    return HttpResponse(content, content_type="text/plain")
```

Comment:
This suggestion includes a couple things.

First, the removal of the `cache_page` decorator. Since this is returning a simple string, and making no database calls, introducing caching here has no payoff. In fact, it's probably _worse_ to use caching here, as we use the database as a cache backend, so it's actually making a database call where it usually wouldn't.

Second, since triple quote strings preserve whitespace, the webpage content was indented by 4 spaces. The suggestion removes that, putting it in line with a normal `robots.txt`. I also included a matching suggestion for the test.
```suggestion
from __future__ import annotations

from django.http import HttpResponse


def robots_txt_view(request):
    content = 'User-agent: *\nDisallow: /'
    return HttpResponse(content, content_type='text/plain')
```